### DISCUSSION TITLE
MainViewModel constructor is called twice

### DISCUSSION THREAD

**Discussion by HowardPWeiss** [unoplatform/uno#16778] [Q&A]
LInux Desktop application

I have a method to create my MainViewModel.

This method gets called twice when my application starts.  Why is this?

**Comment by agneszitte** [comment]
cc @jeromelaban, @Youssef1313

**Reply by jeromelaban** [reply]
@HowardPWeiss can you detail how your viewmodel is build? Are you using MVUX or other? A sample will help troubleshoot.
